@here/harp-mapview/lib/MapView.ts

```diff
@@ -546,6 +552,22 @@ export interface MapViewOptions {
      */
     backgroundTilingScheme?: TilingScheme;
 
+    /**
+     * Should be the [[PolarTileDataSource]] used on spherical projection.
+     */
+    enablePolarDataSource?: boolean;
+
+    /**
+     * The name of the [[StyleSet]] used by [[PolarTileDataSource]] to evaluate for the decoding.
+     */
+    polarStyleSetName?: string;
```

Comment:
what's the default?

Reply:
there is `const DEFAULT_POLAR_STYLE_SET_NAME = "polar";` above

Reply:
added comment